src/08-accessing.js

```diff
@@ -148,7 +166,9 @@ function getCityAndState(person) {}
  *  getFullAddress(person);
  *  //> "697 Pine Drive 2A, Staten Island, NY, 10306"
  */
-function getFullAddress(person) {}
+function getFullAddress(person) {
+  return `${person.address.street} ${person.address.unit}, ${person.address.city}, ${person.address.state}, ${person.address.zip}`
```

Comment:
Remember that with template strings using backticks we have the ability to make our strings exist on multiple lines.  When a string is as long as line 170 we should consider spreading it out among many lines.